app/domain/search_strategies/sql.rb

```diff
@@ -46,8 +46,11 @@ class Sql < Base
                 "phone_numbers.contactable_type = 'Group'"]
       },
       'Event' => {
-        attrs: ['events.name', 'events.number', 'groups.name'],
-        joins: [:groups]
+        attrs: ['event_translations.name', 'events.number', 'groups.name'],
+        joins: ['LEFT JOIN event_translations ON event_translations.event_id = events.id',
+                'INNER JOIN events_groups ON events.id = events_groups.event_id',
+                "INNER JOIN #{Group.quoted_table_name} ON " \
+                "events_groups.group_id = #{Group.quoted_table_name}.id"]
```

Comment:
Weshalb ist es neu nötig, die Gruppen so ausführlich zu joinen? Warum geht `joins: ['LEFT JOIN event_translations ON event_translations.event_id = events.id', :groups]` nicht?

Reply:
Ich hatte das auch probiert, weil ich eigentlich nicht so SQL schreiben möchte, leider ohne Erfolg. Ich vermute, dass man explizites Join-SQL und symbole nicht mischen kann. Ich bin aber nicht in die Tiefe gegangen. Wenn das nicht so abstrahiert wäre, würde ich gerne nur `Event.with_translations.joins(:groups)` sagen. Aber wie gesagt: ich habe die Abkürzung genommen, die funktioniert hat.